scripts/protocgen.sh

```diff
@@ -26,7 +26,7 @@ protoc_install_gocosmos() {
   # So the command below issues a warning and we are muting it for now.
   #
   # Installing plugins must be done outside of the module
-  (cd ~; go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@v0.3.1 2> /dev/null)
```

Comment:
why this was a problem?

Reply:
cause this doesn't generate the code using go cosmos, then breaking all the generated code. I not sure if this worked for you for local generation but the scripts and proto generation is meant to be used with docker to avoid needing to install all the deps.